fli-iam/shanoir-ng
# shanoir-issue#2482: call subject delete in preclinical and datasets at the same time

How to test:
Create new study
Create new subject and add it to the new study
Open study details > Edit > Subject tab > Remove subject > Save study
Subject should be removed from the study without error
If the subject do not belong to any other study, it should be deleted along with all of its data (examinations, acquisitions, datasets)

Also try with a **subject that has examinations** and a **preclinical subject**
